ntbs-service/Startup.cs

```diff
@@ -27,6 +29,7 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Newtonsoft.Json;
```

Comment:
I don't quite understand what is going on here. Are we replacing `Newtonsoft.Json` with `System.Text.Json`? If so, do we still need it to be around, or even as a dependency in our project?

Reply:
It's used in a few other places that isn't auditing. I don't think we planned on switching every single bit over but I'll check with Josh

Reply:
It might be a dumb question but: why do we need to change this over? If we're still using Newtonsoft here, why not for auditing as well?

Reply:
Yep, the plan is to make a tech debt ticket for the full switch

Reply:
Basically the issue in the end was trying to use the default settings from Audit.Core: Audit.Core.Configuration.JsonSettings. Previously the type of this (JsonSerializerOptions) was assignable to Newsoft's type for their options: JsonSerializerSettings. But now that is not the case. 
So we decided that as the .NET standard is now System.Type.Json we should switch over.

Reply:
Thanks for explaining that, it makes sense now!